src/iap/modules/auth/services/auth-forms-api.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@angular/core';
+
+import { LoginForm, SignupForm } from '@shared/models/auth.models';
+import { UserService } from '@shared/modules/api/services/user.service';
+import { JwtAuthService } from '@shared/services/jwt-auth.service';
+import { switchMap } from 'rxjs';
```

Comment:
:point_right: :point_left: 
```suggestion
import { Injectable } from '@angular/core';

import { switchMap } from 'rxjs';

import { LoginForm, SignupForm } from '@shared/models/auth.models';
import { UserService } from '@shared/modules/api/services/user.service';
import { JwtAuthService } from '@shared/services/jwt-auth.service';
```